lib/rack/response.rb

```diff
@@ -128,6 +128,7 @@ def not_found?;           status == 404;                        end
       def method_not_allowed?;  status == 405;                        end
       def precondition_failed?; status == 412;                        end
       def unprocessable?;       status == 422;                        end
+      def unavailable?;         status == 451;                        end
```

Comment:
Omitting is probably OK. I imagine use of it would be sufficiently rare that the code could be checked directly.

However, for completion, it might be a good idea to keep it. Maybe call it `legally_unavailable?` or `unavailable_legally?` or `unavailable_by_law?` or `censored?` or `censored_by_law?`